package.json

```diff
@@ -82,10 +82,8 @@
     },
     "resolutions": {
         "//": [
-            "jasmine-core: Freezing jasmine version used in Karma test suite. jasmine-core@4 drops support for IE11.",
-            "iltorb: Native module used by bundlesize which slows down yarn install and may fail on some envs. Not needed."
+            "jasmine-core: Freezing jasmine version used in Karma test suite. jasmine-core@4 drops support for IE11."
         ],
-        "jasmine-core": "3.10.1",
-        "iltorb": "npm:@stdlib/utils-noop@0.0.10"
+        "jasmine-core": "3.10.1"
```

Comment:
I removed this `iltorb` hack because it's too hard to maintain. It breaks every time you run `yarn upgrade-interactive`. Also this was just a hack to make `yarn install` faster.

The long-term solution is to remove `bundlesize` entirely because it seems unmaintained. I'll do that separately.

Reply:
tada: https://github.com/salesforce/lwc/pull/3146